phpBB/phpbb/db/migration/data/v32x/user_notifications_table_reduce_column_sizes.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v32x;
+
+class user_notifications_table_reduce_column_sizes extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array(
+			'\phpbb\db\migration\data\v32x\user_notifications_table_remove_duplicates',
+		);
+	}
+
+	public function update_schema()
+	{
+		return array(
+			'change_columns'	=> array(
+				$this->table_prefix . 'user_notifications'			=> array(
+					'item_type'		=> array('VCHAR:155', ''),
```

Comment:
I went by what is stated [here](https://dev.mysql.com/doc/refman/5.7/en/string-type-overview.html):

> utf8 characters can require up to three bytes per character, so a VARCHAR column that uses the utf8 character set can be declared to be a maximum of 21,844 characters

Based on that and taking into account the 1000 max byte count error I just used simple math:
1000 bytes / 3 bytes per char = 333.33 chars / 2 columns = 166.67 chars per column

155 seemed a little more "common" a size when compared to other columns. I can't imagine either of these columns having data being even remotely close to that amount so that's what I went with.

tldr; I can probably go as high as `VARCHAR:166` based on MySQL docs so up to you if you want me to use that amount.